Vic2ToHoI4/Source/OutHoi4/Map/OutSupplyNodes.cpp

```diff
@@ -0,0 +1,21 @@
+#include "OutSupplyNodes.h"
+
+#include <fstream>
+
+
+
+void HoI4::outputSupplyNodes(const std::string& path, const std::set<int>& SupplyNodes)
+{
+	std::ofstream out(path + "/map/supply_nodes.txt");
+	if (!out.is_open())
+	{
+		throw std::runtime_error("Could not open " + path + "/map/supply_nodes.txt");
+	}
+
+	for (const auto& supplyNode: SupplyNodes)
+	{
+		out << "1 " << supplyNode << "\n";
```

Comment:
What is 1? Something default?

Reply:
Yep, every line in vanilla HoI4 is preceded by this 1. I'd guess early in the design supply nodes could have been several levels.